src/zope/sqlalchemy/tests.py

```diff
@@ -29,8 +29,11 @@
 import time
 import unittest
 
+from pkg_resources import parse_version
```

Comment:
Cloud you please replace `parse_version` with `packaging.version.Version` as it is just a redirect and `pkg_resources` is deprecated nowadays.

See https://github.com/pypa/setuptools/blob/e94cb204693cee145ea531905dd50b02a26c9f0a/pkg_resources/__init__.py#L134 and https://setuptools.pypa.io/en/latest/pkg_resources.html

Reply:
Sure, I followed the template from `datamanager.py`, which did it the same way but I can update that as well.